src/ocr.py

```diff
@@ -7,10 +7,11 @@
 import json
 from difflib import SequenceMatcher
 from operator import itemgetter
-
+import sys
+# TODO find logging library so we can set it to error,warn,info,debug levels
 
 class CardImage:
-	def __init__(self, filename):
+	def __init__(self, filename, show_crops, do_ocr):
```

Comment:
Tests are breaking because they expect the constructor to accept only two parameters. Let's update this so they have defaults specified.